### ISSUE TITLE
Panels dimensions are still not being rendered correctly

### ISSUE THREAD

**Issue by hafen** [trelliscope/trelliscopejs-lib#741]
A display will always have a `panelaspect` property, which is the width divided by the height of the panel.

A rendered panel always needs to conform to the following rules:

1. The size of the panel must always have this aspect ratio
2. The image inside of the panel must preserve its original aspect ratio

There are a few issues here that need to be addressed:

## Need to add `object-fit: contain` to img tags

As an example, Pokemon display currently does conform to rule 2. For example, an image in the panel looks like this:

<img width="836" alt="image" src="https://user-images.githubusercontent.com/1275592/223235012-447138b1-0015-4628-848e-525ce601706a.png">

While the actual image looks like this:

<img width="469" alt="image" src="https://user-images.githubusercontent.com/1275592/223235061-abc4c4b3-6526-4c2c-b7e9-2aa70fe80e9a.png">

Even though the specified aspect ratio is 1.5, that only means that the actual panel should have this aspect ratio. The image being shown inside of it must preserve its original aspect ratio.

Basically, all you need to do is add `object-fit: contain` to the `img` tag and you should get something like this, which is the desired outcome:

<img width="838" alt="image" src="https://user-images.githubusercontent.com/1275592/223236314-c8a03a4c-fd0f-48ff-ab1c-e67da65249a6.png">

## Safari is still not preserving aspect ratio when changing number of columns

If you open up the Pokemon example in Safari, which has a default of 6 columns, and then reduce the number of columns to 1, you get the following:

<img width="1727" alt="image" src="https://user-images.githubusercontent.com/1275592/223236795-e0e7c9b2-08d1-4a41-8f7a-73f19cd97cfe.png">

This isn't happening in Chrome.

The same issue happens with iframe panels, such as the network_nonraster example.
